app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -348,11 +360,11 @@ public void onResponse(Call<MemoryResponse> call, Response<MemoryResponse> respo
                                 isHavingLink = urlList != null;
                                 if (urlList.size() == 0) isHavingLink = false;
 
-                                updateDatabase(c, query, DateTimeHelper.getDate(), false, true, false, false, false, false, isHavingLink, DateTimeHelper.getCurrentTime(), false, null);
+                                updateDatabase(c, query, DateTimeHelper.getDate(), false, true, false, false, false, false, isHavingLink, DateTimeHelper.getCurrentTime(), false, null, "", "", "");
                                 parseSusiResponse(allMessages.get(i));
                                 rvChatFeed.getRecycledViewPool().clear();
                                 recyclerAdapter.notifyItemChanged((int) c);
-                                updateDatabase(c + 1, answer, DateTimeHelper.getDate(), false, false, isSearchResult, isWebSearch, false, isMap, isHavingLink, DateTimeHelper.getCurrentTime(), isPieChart, datumList);
+                                updateDatabase(c + 1, answer, DateTimeHelper.getDate(), false, false, isSearchResult, isWebSearch, false, isMap, isHavingLink, DateTimeHelper.getCurrentTime(), isPieChart, datumList, maplatitude, maplongitude, zoom);
```

Comment:
Instead of storing it directly pls make a model of type map and store it in, Only in the case where the answer is map.